include/openssl/x509_vfy.h

```diff
@@ -558,7 +558,7 @@ int X509_VERIFY_PARAM_set_flags(X509_VERIFY_PARAM *param,
                                 unsigned long flags);
 int X509_VERIFY_PARAM_clear_flags(X509_VERIFY_PARAM *param,
                                   unsigned long flags);
-unsigned long X509_VERIFY_PARAM_get_flags(X509_VERIFY_PARAM *param);
+unsigned long X509_VERIFY_PARAM_get_flags(const X509_VERIFY_PARAM *param);
```

Comment:
Probably this part of the change should be backported. Maybe in a different PR.

Reply:
I've tentatively started constifying many functions in `x509_vfy.h`, indeed better in a separate PR...

Reply:
Done, see #10504